style.css

```diff
@@ -46,46 +55,142 @@ body {
 
 main {
     width: 100%;
+    margin-left: 40px;
 }
 
 .description-box {
-    width: 70%;
-    margin-left: 200px;
-
+    display: flex;
+    justify-content: center;
+    margin: 30px 120px 30px 120px;
 }
 
-aside {
-    width: 450px;
+.description-text {
+    width: 100%;
+    font-weight: 500;
+    font-size: 20px;
+    text-align: justify;
 }
 
-.header-second {
-    font-size: 17px;
+.experience-title {
+    font-size: 25px;
+    margin: 25px 0 25px 0;
 }
 
-.job-first {
+.work-experience-part-one {
     width: 100%;
+    margin-bottom: 12px;
+    border-bottom: 1px solid rgba(0, 0, 0, 0.1);
 }
 
-.job-first-description {
-    width: 25%
+.work-experience-part-one-description {
+    width: 45%;
+    font-size: 20px;
 }
 
-.job-first-list {
-    width: 70%;
+.work-experience-part-one-list {
+    width: 100%;
+    margin-top: 20px;
     margin-left: -24px;
-    margin-bottom: 45px;
 }
 
 .highlights-list {
     color: #35a5f6;
     font-weight: bold;
 }
 
-.job-second-description {
-    width: 25%
+.work-experience-part-two {
+    margin-bottom: 10px;
+}
+
+.work-experience-part-two-description {
+    width: 39%;
+    font-size: 20px;
 }
 
-.job-second-list {
-    width: 70%;
+.work-experience-part-two-list {
+    width: 100%;
+    margin-top: 20px;
     margin-left: -24px;
 }
+
+.list-item {
+    width: 97%;
+    font-size: 15px;
+    margin: 10px 0 10px 0;
+    text-align: justify;
+}
+
+.contact-education-skills-box {
+    width: 50%;
+}
+
+.contact-title {
+    font-size: 20px;
+    margin: 30px 0 10px 0;
+}
+
+.phone-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.phone-details {
+    margin: 10px 0 15px 10px;
+}
+
+.mail-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.mail-details {
+    margin: 10px 0 15px 10px;
+}
+
+.linkedin-box {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+
+.linkedin-details {
+    margin: 10px 0 15px 10px;
+}
+
+.education-title {
+    font-size: 20px;
+    margin: 25px 0 15px 0;
+}
+
+.education-item-details {
+    font-weight: bold;
+}
+
+.relevant {
+    margin-bottom: 15px;
+}
+
+.relevant-skills-title {
+    font-size: 20px;
+    margin: 30px 0 25px 0;
+}
+
+.relevant-skills-item {
+    margin-left: 1px;
+}
+
+.footer-box {
+    display: flex;
+    justify-content: center;
+    margin: 25px 120px 30px 120px;
+}
+
+.footer-text {
+    width: 100%;
+    font-size: 12px;
+    color: rgba(0, 0, 0, 0.5);
```

Comment:
Here, you are providing the third argument rgba - 0.5.
It means that the color is black, but it is 50% transparent. 

We don't really need transparency here. A more straightforward approach would be to provide an exact gray color instead of using transparency. It would also mean that the text color would stay the same even if the background changes.